dbbackup/management/commands/dbbackup.py

```diff
@@ -43,16 +45,29 @@ def handle(self, **options):
         self.compress = options.get('compress')
         self.encrypt = options.get('encrypt')
         self.storage = BaseStorage.storage_factory()
-        database_keys = (self.database,) if self.database else dbbackup_settings.DATABASES
-        for database_key in database_keys:
-            database = settings.DATABASES[database_key]
-            self.dbcommands = DBCommands(database)
+        if options.get('mongo'):
+            # Mongo specific handling.
+            # TODO: Add support for multiple mongo db defined in settings.
```

Comment:
I always thought that if ENGINE doesn't exists django would complain, but apparently it does not...
I'll make the necessary changes.